test/worker_test.py

```diff
@@ -1774,7 +1793,7 @@ def requires(self):
 
                     self.assertTrue(w3.run())
                     self.assertFalse(w2.run())
-                    self.assertFalse(w1.run())
+                    self.assertTrue(w1.run())
```

Comment:
I'm actually not sure what happened here. I was planning to look into it but forgot.